openapi/README.rst

```diff
@@ -9,32 +9,65 @@ d.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
 Set up REST API and export Open API (Swagger) specification file for
 integration with whatever you need.
 
-The module handles routes prefixed with ``/api/``. 
+This module impements a ``/api/v1/`` route tree. 
 
 Authentication
 ==============
 
+* Server wide "break glass" through a ``salt`` in the config file
+* Database inference: Database name encoded and appended to the token
+* User authentication through the actual token
+
 As a workaround for multi-db instances, system uses *Basic Authentication* with
 ``db_name:token`` credentials, where ``token`` is a new field in ``res.users``
 model.
 
 Customization
 =============
 
+TODO
+
+The model already includes configuration for the following models:
+
+  * report
+  * sale.order
+  * sale.order.line
+  * account.invoice
+  * account.invoice.line
+  * res.partner
+  * product.template
+
+TODO 
+
+If you install those models after installing this module,
+you need to upgrade this module so that the new defaults are loaded.
+
+Reason: We check installed modules dynamically so to only load defaults of existing models.
+
 The module allows to configure
 
 * available models
-* available operations per model (GRUD)
+* available operations per model (CRUD)
+* enable/disable private methods through API
+* method whitelist
+* method blacklist
+* specification for the following operations
+
+  * For reading ONE:
+
+    * return field sets
 
-  * For reading:
+  * For reading MULTI:
 
-    * field sets
+    * return field sets
+    * default domain
 
 .. TODO: add example of usage in API requests
 
   * For creation:
 
     * default values
+    * return fields set
```

Comment:
We can discuss this, but it might be useful to obtain some fields, like computed fields, to augment origin records with odoo provided values. A subsequent GET to read enpoint would do as well, obviously...